files/default/neutron-ha-tool.py

```diff
@@ -237,9 +237,9 @@ def run(args):
     qclient.format = 'json'
 
     if args.agent_selection_mode == 'random':
-        agent_picker_class = RandomAgentPicker
+        agent_picker = RandomAgentPicker()
```

Comment:
You are right, clients mean the clients of the AgentPicker which is confusing, as you pointed it out. Let me re-phrase this.